test/common.js

```diff
@@ -497,6 +497,12 @@ function fail(msg) {
 }
 exports.fail = fail;
 
+exports.mustNotCall = function(msg) {
+  return function dontCall() {
```

Comment:
Would it help to keep this function's name also as `mustNotCall`? Stacktrace would be better I guess.